### PR TITLE
KOR : Apply #506, #511

### PR DESCRIPTION
**Pull request by telk5093**

### CODE REVIEW

**Comment by Nubbiy** [comment]
See also issue #511 
More strings added just now

**Comment by telk5093** [comment]
@Nubbiy Done. Thanks for noticing me.

File: data/language/korean.txt [hunk 1]
```diff
@@ -4092,6 +4092,17 @@ STR_5786    :유효하지 않은 그룹 이름
...
+STR_5791    :{SMALLFONT}{BLACK}모든 놀이기구의 신뢰도를 100%로 설정하고{NEWLINE}건설 시점을 "올해"로 바꿉니다.
+STR_5792    :{SMALLFONT}{BLACK}고장난 놀이기구를 모두 고칩니다.
+STR_5793    :{SMALLFONT}{BLACK}놀이기구의 충돌 이력을 제거하여,{NEWLINE}손님들이 안전하지 않다고 불평하지 않게 만듭니다.
+STR_5794    :{SMALLFONT}{BLACK}일부 시나리오에서 공원에 이미 존재하는 놀이기구를 수정할 수 없게 해놓은 것을 해제해줍니다.
+STR_5795    :{SMALLFONT}{BLACK}손님들이 놀이기구의 격렬도가 아무리 높아도 그 놀이기구에 타도록 만듭니다.
+STR_5796    :{SMALLFONT}{BLACK}공원의 문을 강제로 열거나 닫습니다.
+STR_5797    :{SMALLFONT}{BLACK}날씨를 선택한 날씨로 고정하고 바뀌지 않게 만듭니다.
+STR_5798    :{SMALLFONT}{BLACK}일시정지 상태에서 건설을 허용합니다.
+STR_5799    :{SMALLFONT}{BLACK}브레이크 실패로 인한 고장과 충돌이 일어나지 않게 만듭니다.
+STR_5780    :{SMALLFONT}{BLACK}놀이기구가 고장나지 않게 만듭니다.
```
Comment (Nubbiy) [inline data/language/korean.txt]: This string already exsist and should be changed to STR_5800 
Really sorry about the inconveniences

Comment (telk5093) [inline data/language/korean.txt]: Fixed. No problem ;)

**Pull request merged**